file_test.go

```diff
@@ -40,8 +52,30 @@ func TestDownloadGitTagZipFile(t *testing.T) {
 			GitTag: tc.gitTag,
 		}
 
-		zipFilePath, err := downloadGithubZipFile(gitHubCommit, tc.githubToken)
+		zipFilePath, err := downloadGithubZipFile(gitHubCommit, tc.githubToken, tc.instance)
+
 		defer os.RemoveAll(zipFilePath)
+
+		// We don't have a running instance of GitHub Enterprise against which to validate tests as we do for GitHub public,
+		// so this test will only validate that fetch attempted to download from the expected URL. The download itself
+		// will fail.
+
+		githubEnterpriseDownloadUrl := fmt.Sprintf("https://%s/repos/%s/%s/zipball/%s", tc.instance.ApiUrl, tc.repoOwner, tc.repoName, tc.gitTag)
+
+		// TODO: The awkwardness of this test makes it clear that a better structure for this program would be to refactor
+		// the downloadGithubZipFile() function to a function called downloadGithubFile() that would accept a URL as a
+		// param. We could then test explicitly that the URL is as expected, which would make GitHub Enterprise test cases
+		// simpler to handle.
+
+		if err != nil && strings.Contains(err.Error(), "no such host") {
+			if strings.Contains(err.Error(), githubEnterpriseDownloadUrl) {
+				t.Logf("Found expected download URL %s. Download itself failed as expected because no GitHub Enterprise instance exists at the given URL.", githubEnterpriseDownloadUrl)
+				t.SkipNow()
```

Comment:
Does it work if we `return` here to end the execution with a `PASS` instead of being marked as `SKIP`?

Reply:
I think so. Unfortunately I'm being throttled by GitHub's API so I've committed this for now in 706117f853a8ae7fad577f4ad016a6550747c7ff. Let's see how Circle tests look.